src/ripple/app/consensus/LedgerConsensus.cpp

```diff
@@ -1720,10 +1720,10 @@ class LedgerConsensusImp
     */
     void playbackProposals ()
     {
-        for (auto it: getApp().getOPs ().peekStoredProposals ())
+        for (auto const& it: getApp().getOPs ().peekStoredProposals ())
```

Comment:
Hi Howard, just checking.  It looks like previously `it` was making a copy of a `std::list<shared_ptr<LedgerProposal>>`.  Since we're now taking a reference to the `list`, those `shared_ptr`s are not having their reference counts incremented for the duration of the operation.

I'm guessing that's nothing to worry about because the previous `list` copy was not happening under a lock.  That means the `list` must be stable in this context (or there's a bug).  As long as the `list` is stable the `shared_ptr`s it contains must have a reference count of at least  1.  So the `shared_ptr`s are okay.  I just had to  talk myself through that...